src/views/Package/components/PackageDocs/PackageDocs.tsx

```diff
@@ -20,12 +20,15 @@ export const appendItem = (itemTree: Item[], item: Element): Item[] => {
 
   const { headingId, headingTitle = "", headingLevel = "100" } = item.dataset;
   const level = parseInt(headingLevel);
-  if (level > 3) {
+
+  // Don't create nav items for items with no title / url
+  if (level > 3 || !headingTitle || !headingId) {
```

Comment:
Occasionally some navItems are expandable, but have no items. Checking for the presence of a url or title fixes this